addons/html_builder/static/src/core/core_builder_action_plugin.js

```diff
@@ -259,36 +259,25 @@ export class StyleAction extends BuilderAction {
         return currentValue === value;
     }
     apply({ editingElement, params = {}, value }) {
+        if (!this.delegateTo("apply_custom_css_style", { editingElement, params, value })) {
+            this.applyCustomCssStyle({ editingElement, params, value });
```

Comment:
For the naming, IMO, either you do `this.delegateTo("apply_css_style")` with the `apply_css_style` defined in `core_builder_action_plugin.js` and `color_style_plugin.js` either you do `this.delegateTo("apply_custom_css_style")` with `apply_custom_css_style` only defined in `color_style_plugin.js` but then you can rename `this.applyCustomCssStyle` -> `this.applyCssStyle`